android/julius/src/main/java/bvschaik/julius/JuliusMainActivity.java

```diff
@@ -0,0 +1,120 @@
+package bvschaik.julius;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.widget.Toast;
+import org.libsdl.app.SDLActivity;
+
+public class JuliusMainActivity extends SDLActivity
+{
+    private static final int GET_FOLDER_RESULT = 500;
+    private static final int rwFlagsPermission = Intent.FLAG_GRANT_READ_URI_PERMISSION
+                                               | Intent.FLAG_GRANT_WRITE_URI_PERMISSION;
+    public static boolean paused = true;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState)
+    {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public void onStart()
+    {
+        super.onStart();
+        paused = false;
```

Comment:
Either use onStart/onStop for setting paused, or use onResume/onPause (I think this one is preferred here). Don't mix both.